src/views/common/index/home.njk

```diff
@@ -86,7 +86,7 @@
       </svg>
     </button>
     <p>
-      <a href="guidance-ACSPs" data-event-id="Read the guidence">{{ i18n.readThe }}</a>.
+      <a href="https://www.gov.uk/guidance/register-as-an-authorised-agent" data-event-id="Read the guidence">{{ i18n.readThe }}</a>.
```

Comment:
Its saying page not found for this link. Is it expected ?

Reply:
Yes, the content has not been added yet